python/ray/serve/tests/test_pipeline_dag.py

```diff
@@ -129,7 +129,7 @@ def test_single_func_no_input(serve_instance):
 
 def test_single_func_deployment_dag(serve_instance):
     with InputNode() as dag_input:
-        dag = combine.bind(dag_input[0], dag_input[1], kwargs_output=1)
+        dag = combine.bind().bind(dag_input[0], dag_input[1], kwargs_output=1)
```

Comment:
i didn't quite see why this is better UX than current test. combine as a function, when called on `.bind()` returns a FunctionNode that takes the original function signature.

1) it diverges from ray dag syntax 
2) we need to now use `.bind().bind()` for every function in DAG ?

If the point is to support `handle = serve.run(func.bind())` without overloading existing `DeploymentNode` UX that returns either class or function, I think the right thing is to introduce a `DeploymentFunctionNode` that maps to `FunctionNode` in ray.